service/provider/src/main/java/gov/va/vro/service/provider/mas/service/mapper/MasCollectionAnnotsResults.java

```diff
@@ -123,7 +132,7 @@ private static AbdMedication createMedication(
     if (masAnnotation.getObservationDate() != null) {
       abdMedication.setAuthoredOn(masAnnotation.getObservationDate().replaceAll("Z", ""));
     } else {
-      abdMedication.setAuthoredOn("9999-12-31");
+      abdMedication.setAuthoredOn(NOT_AVAILABLE_DT);
```

Comment:
`authoredOn` from lighthouse has the longer format datetime. Example: 2022-03-23T01:43:31Z. See `resources/test-7101-01/assessment.json`